db/migrate/20211203233820_create_posts.rb

```diff
@@ -0,0 +1,16 @@
+class CreatePosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+      t.bigint :author_id
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
+
+    add_foreign_key :posts, :users, column: :author_id
+    add_index :posts, :author_id
```

Comment:
- [ ] Timestamp of this migrations is before the `CreateUsers` migration. So this file will run before the user table is created. As a result of that, it will throw an error saying that the relation is not created because the `users` table doesn't exist

**If you need to alter a table, you should create a  new migration instead of editing the old one.** 
You may not have seen this error if you already have the users table when you edited this migration file. Please drop all the tables and try to run migration from scratch.

![Screenshot from 2021-12-04 15-57-32](https://user-images.githubusercontent.com/1642014/144710265-0bf1f37c-91a4-452f-8e39-ea9a2cbbbfea.png)